portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,6 +37,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String email = request.getParameter("email-input");
         String subject = request.getParameter("subject-input");
         String message = request.getParameter("message-input");
+        long timestamp = System.currentTimeMillis();
 
         // Error handling - don't allow empty or null values
         if (name != null && email != null && subject != null && message != null) {
```

Comment:
Also do you want to be this strict? If so, what sort of UI feedback would the user get if one of these values is null or empty so that they know that their comment submission was invalid?